src/cmd/run.js

```diff
@@ -9,24 +12,18 @@ import {
 import {createLogger} from '../util/logger';
 import getValidatedManifest, {getManifestId} from '../util/manifest';
 import defaultSourceWatcher from '../watcher';
-
-
-const log = createLogger(__filename);
-
-
-// Import objects that are only used as Flow types.
-import type FirefoxProfile from 'firefox-profile';
 import type {OnSourceChangeFn} from '../watcher';
-import type Watchpack from 'watchpack';
 import type {
-  FirefoxProcess,
+  FirefoxProcess, //eslint-disable-line import/named
```

Comment:
aww, it doesn't work with the flow linter? That's too bad. This is a good way to workaround it though.